### PR TITLE
Add changelogs to most things in the DB

### PR DESCRIPTION
**Pull request by apmasell**
Samples are the case study and then cookie-cuttered out for other object types.

### CODE REVIEW

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractExperiment.java [hunk 1]
```diff
@@ -38,329 +39,345 @@
...
-   public Study getStudy() {
-      return study;
-   }
-
-   @Override
-   public void setStudy(Study study) {
-      this.study = study;
-   }
-
-   @CoverageIgnore
```
Comment (frankie13) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractExperiment.java]: Apparently you can use cobertura `ignoreTrivial` in the pom to ignore straight getters and setters.
I have never gotten it to work properly with maven though so this seems like the best solution :white_check_mark:

**Comment by frankie13** [comment]
When I first start this in a browser I get this stacktrace:

ERROR [http-bio-8080-exec-11] - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'daoLookup' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'emPCRStore' while setting bean property 'daos' with key [TypedStringValue: value [uk.ac.bbsrc.tgac.miso.core.data.impl.emPCR], target type [class java.lang.Class]]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'emPCRStore': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlEmPCRDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlLibraryDilutionDAO' while setting bean property 'libraryDilutionDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlLibraryDilutionDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlLibraryDAO' while setting bean property 'libraryDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlLibraryDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'libraryNamingScheme' defined in ServletContext resource [/WEB-INF/miso-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestManager' defined in ServletContext resource [/WEB-INF/miso-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'experimentStore': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlExperimentDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlStudyDAO' while setting bean property 'studyDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlStudyDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlProjectDAO' while setting bean property 'projectDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlProjectDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlRunDAO' while setting bean property 'runDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlRunDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlSequencerPartitionContainerDAO' while setting bean property 'sequencerPartitionContainerDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlSequencerPartitionContainerDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlPartitionDAO' while setting bean property 'partitionDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlPartitionDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlPoolDAO' while setting bean property 'poolDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlPoolDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'securityDAO' of bean class [uk.ac.bbsrc.tgac.miso.sqlstore.SQLPoolDAO]: Bean property 'securityDAO' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:329)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:107)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedMap(BeanDefinitionValueResolver.java:379)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:162)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1360)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1118)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:456)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:294)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:225)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:291)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:605)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:925)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:472)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:383)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:283)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5017)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5531)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.manager.ManagerServlet.start(ManagerServlet.java:1322)
	at org.apache.catalina.manager.HTMLManagerServlet.start(HTMLManagerServlet.java:694)
	at org.apache.catalina.manager.HTMLManagerServlet.doPost(HTMLManagerServlet.java:217)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.filters.CsrfPreventionFilter.doFilter(CsrfPreventionFilter.java:213)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.filters.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:108)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:614)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:957)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:620)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)

**Comment by frankie13** [comment]
When I start the app I get this stacktrace.  Am I missing a migration maybe?:

ERROR [http-bio-8080-exec-11] - Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'daoLookup' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'emPCRStore' while setting bean property 'daos' with key [TypedStringValue: value [uk.ac.bbsrc.tgac.miso.core.data.impl.emPCR], target type [class java.lang.Class]]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'emPCRStore': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlEmPCRDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlLibraryDilutionDAO' while setting bean property 'libraryDilutionDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlLibraryDilutionDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlLibraryDAO' while setting bean property 'libraryDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlLibraryDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'libraryNamingScheme' defined in ServletContext resource [/WEB-INF/miso-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestManager' defined in ServletContext resource [/WEB-INF/miso-config.xml]: Initialization of bean failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'experimentStore': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlExperimentDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlStudyDAO' while setting bean property 'studyDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlStudyDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlProjectDAO' while setting bean property 'projectDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlProjectDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlRunDAO' while setting bean property 'runDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlRunDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlSequencerPartitionContainerDAO' while setting bean property 'sequencerPartitionContainerDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlSequencerPartitionContainerDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlPartitionDAO' while setting bean property 'partitionDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlPartitionDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Cannot resolve reference to bean 'sqlPoolDAO' while setting bean property 'poolDAO'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'sqlPoolDAO' defined in ServletContext resource [/WEB-INF/db-config.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'securityDAO' of bean class [uk.ac.bbsrc.tgac.miso.sqlstore.SQLPoolDAO]: Bean property 'securityDAO' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:329)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:107)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedMap(BeanDefinitionValueResolver.java:379)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:162)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1360)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1118)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:456)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:294)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:225)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:291)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:193)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:605)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:925)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:472)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:383)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:283)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:111)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5017)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5531)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.manager.ManagerServlet.start(ManagerServlet.java:1322)
	at org.apache.catalina.manager.HTMLManagerServlet.start(HTMLManagerServlet.java:694)
	at org.apache.catalina.manager.HTMLManagerServlet.doPost(HTMLManagerServlet.java:217)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:650)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.filters.CsrfPreventionFilter.doFilter(CsrfPreventionFilter.java:213)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.filters.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:108)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:614)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:957)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:620)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)

**Comment by apmasell** [comment]
Sorry, I had a bad rebase and I didn't realise. :-(

**Comment by frankie13** [comment]
I have run the migrations and I still get a huge stack trace when I try and start miso.
I double checked the database and migration seems to have run ok.
Does it do this for you @apmasell ??
here's a chunk:

	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:90)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)
	at uk.ac.bbsrc.tgac.miso.core.manager.UserAuthMisoRequestManager$$EnhancerByCGLIB$$f6b2b00c.listAllRuns(<generated>)
	at uk.ac.bbsrc.tgac.miso.webapp.context.MisoAppListener.contextInitialized(MisoAppListener.java:281)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:5017)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5531)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:901)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:877)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:652)
	at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:677)
	at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1912)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
java.sql.SQLException: Column 'lastModifier' not found.
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:926)
	at com.mysql.jdbc.ResultSetImpl.findColumn(ResultSetImpl.java:1144)
	at com.mysql.jdbc.ResultSetImpl.getLong(ResultSetImpl.java:3001)
	at uk.ac.bbsrc.tgac.miso.sqlstore.SQLRunDAO$RunMapper.mapRow(SQLRunDAO.java:719)
	at uk.ac.bbsrc.tgac.miso.sqlstore.SQLRunDAO$RunMapper.mapRow(SQLRunDAO.java:665)
	at org.springframework.jdbc.core.RowMapperResultSetExtractor.extractData(RowMapperResultSetExtractor.java:92)
	at org.springframework.jdbc.core.RowMapperResultSetExtractor.extractData(RowMapperResultSetExtractor.java:1)
	at org.springframework.jdbc.core.JdbcTemplate$1QueryStatementCallback.doInStatement(JdbcTemplate.java:446)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:396)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:456)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:464)
	at uk.ac.bbsrc.tgac.miso.sqlstore.SQLRunDAO.listAll(SQLRunDAO.java:551)
	at uk.ac.bbsrc.tgac.miso.sqlstore.SQLRunDAO.listAll(SQLRunDAO.java:95)
	at uk.ac.bbsrc.tgac.miso.sqlstore.SQLRunDAO$$FastClassByCGLIB$$860b7395.invoke(<generated>)
	at net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	at com.googlecode.ehcache.annotations.interceptor.EhCacheInterceptor.invokeCacheable(EhCacheInterceptor.java:142)
	at com.googlecode.ehcache.annotations.interceptor.EhCacheInterceptor.invoke(EhCacheInterceptor.java:81)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)

**Comment by frankie13** [comment]
Hi @apmasell - it runs!
I have been looking at it this morning and everything I change seems to be changed by Xingdong, even though I'm logged in as me (see screenshots).  Xingdong is user_id 1 in my database.
Also, why are we showing data that hasn't changed in the log?  Would it be clearer to just show the changes?  Also, it would be good to see some tests on the Controllers / DAO's when we get a framework in place.  I have provided an @CoverageIgnore annotation for trivial methods, but I will be carpet bombing the codebase with that when I start looking at coverage.
prof
sample

**Comment by apmasell** [comment]
The user id 1 thing is a fallback in case the account is not specified.
That must mean there is a bug in the project controller that doesn't update
the user ID, unless you see that everywhere. The n/a → n/a changes are bad
null handling. Again, I thought I fixed those, but apparently not. I'll
have a look.

On 29 October 2015 at 06:50, frankie13 <notifications@github.com> wrote:

> Hi @apmasell <https://github.com/apmasell> - it runs!
> I have been looking at it this morning and everything I change seems to be
> changed by Xingdong, even though I'm logged in as me (see screenshots).
> Xingdong is user_id 1 in my database.
> Also, why are we showing data that hasn't changed in the log? Would it be
> clearer to just show the changes? Also, it would be good to see some tests
> on the Controllers / DAO's when we get a framework in place. I have
> provided an @CoverageIgnore annotation for trivial methods, but I will be
> carpet bombing the codebase with that when I start looking at coverage.
> [image: prof]
> <https://cloud.githubusercontent.com/assets/13831831/10816471/dac40cec-7e2a-11e5-8dce-07ecfb401a96.png>
> [image: sample]
> <https://cloud.githubusercontent.com/assets/13831831/10816470/dac311e8-7e2a-11e5-8f37-a86f0d15b7b9.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TGAC/miso-lims/pull/15#issuecomment-152145352>.
>



-- 
--Andre Masella<andre@masella.name>
http://www.masella.name/

**Comment by apmasell** [comment]
Okay. Hopefully this is less of a disaster. When I rebased, there was no complaints, apparently because it ate everything. You'll have to do `flyway:clean flyway:migrate` and reload your test data.

**Pull request closed without merging**